client/src/components/Q&A/questions.css

```diff
@@ -15,14 +15,14 @@
 }
 
 .search-question {
-  background-color: black;
+  background-color: grey;
```

Comment:
It's a little difficult to read the search question placeholder text here in grey!

Reply:
removed search questions bar and css related to it